tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py

```diff
@@ -48,7 +48,7 @@
             endIndex=7,
             required=True,
             validators=[
-                validators.dateYearIsLargerThan(1998),
+                validators.dateYearIsLargerThan(2019),
```

Comment:
```suggestion
                validators.dateYearIsLargerThan(2020),
```

@atrimpe suggesting 2020 because TDP accepts data for FY2021 forward, which translates to Oct 2020 - Sept 2021. Recommend making this change throughout this ticket.